packages/mendel-middleware/index.js

```diff
@@ -24,6 +24,23 @@ function MendelMiddleware(opts) {
     return function(req, res, next) {
         req.mendel = req.mendel || {};
 
+        req.mendel.getBundleEntries = function() {
+            return Object.keys(trees.bundles).reduce(
```

Comment:
Caching this would not help anything performance wise because this function takes nanoseconds to execute. In the other hand, if we cache it, and a consumer mutates our cache, we are screwed. Computing new arrays and new objects from scratch is safer and it is not slow at all. You can try it, do a 10k loop with this function and it will still be like 1ms in a production server.